pypeit/core/wavecal/templates.py

```diff
@@ -534,6 +534,17 @@ def main(flg):
                        outroot, lowredux=False, ifiles=ifiles, chk=True,
                        normalize=True)
 
+    if flg & (2**21):  # WHT/ISIS
+        reid_path = os.path.join(resource_filename('pypeit', 'data'), 'arc_lines', 'reid_arxiv')
+        iroot = 'wht_isis_blue_1200_4800.json'
+        iout = 'wht_isis_blue_1200_4800.fits'
+        wfile = os.path.join(reid_path, iroot)
+        binspec = 2
+        slits = [0]
+        lcut = [3200.]
+        build_template(wfile, slits, lcut, binspec, iout, lowredux=False)
```

Comment:
Not sure what the 'continuum' is like for this arc, but note that
I've added new bells and whistles in the `load_save_specobjs`
branch to remove the continnum, normalize, etc.